images/copy-images/Dockerfile

```
@@ -0,0 +1,13 @@
+ARG ALPINE_VERSION=3.17
```

Can you pin this to the latest patch version?

```suggestion
ARG ALPINE_VERSION=3.17.2
```